pkgs/applications/version-management/git-branchstack/default.nix

```
@@ -13,7 +13,7 @@ buildPythonApplication rec {
     hash = "sha256-gja93LOcVCQ6l+Cygvsm+3uomvxtvUl6t23GIb/tKyQ=";
   };
 
-  buildInputs = [
+  propagatedBuildInputs = [
```

Now spelled `dependencies`:

```suggestion
  dependencies = [
```

---

(for python packages).

---

https://github.com/NixOS/nixpkgs/pull/348315